playwright/src/main/java/com/microsoft/playwright/impl/APIResponseAssertionsImpl.java

```diff
@@ -37,7 +37,7 @@ public APIResponseAssertionsImpl(APIResponse response) {
   }
 
   @Override
-  public APIResponseAssertions not() {
+  public APIResponseAssertionsImpl not() {
```

Comment:
I change the return value to the Impl for `not` as [@yury-s suggested ](https://github.com/microsoft/playwright-java/pull/1340#discussion_r1291649659)